src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
> This is not my problem. You may want to reflect on the way you interact with other people and the community 

I have done more for this community than most of the people with who I am interacting here will do in their lifetime. You may want to reflect on that too.

> or on the solutions you propose that always try to tie in your company infrastructure in some way.

OPAM was developed on http://github.com/OCamlPro/opam/, the repository hosted on http://github.com/OCamlPro/opam-repository/, and the website hosted on http://opam.ocamlpro.com/. Everything was moved to http://ocaml.org/ That's what you call "always trying to tie in my company" ? The only thing I have fought for was to recover our access rights, since the "community" took all these repos without keeping our access rights to them...

> I don't see an environment partly in control of OCamlPro or any other company as being a sustainable environment

Without Janestreet and OCamlPro, there would be no ocaml.org and no OPAM. Where do you see a sustainable environment without them ? By the way, OCamlPro was created to help the OCaml community, not the other way around.

> The environment should be in control of the community which is what opam.ocaml.org is for, as such I don't feel we should have URIs that point to OCamlPro's infrastructure in the opam source itself.

So, OCamlPro should keep on creating new software and services, then move them to ocaml.org, so that people like you can believe that OCamlPro did not create them, and that they can disrespect us. Nice.